nginx/default.conf

```diff
@@ -0,0 +1,23 @@
+server {
+  listen 80;
```

Comment:
What is the idea of having a proxy?

Reply:
Pros:
We can use the same port for multiple services, and the proxy does the internal routing. We only have to expose one port to the host.

Cons:
It makes the development more complex.

Maybe it's a bit overkill to use one in this project, but I think we can learn more if we do.

These are pros and cons only for development because there are [many](https://medium.com/intrinsic-blog/why-should-i-use-a-reverse-proxy-if-node-js-is-production-ready-5a079408b2ca) reasons why it's better to use a reverse proxy for production.

Reply:
I have used reverse proxies before, +1 from me.